specs/core/0_beacon-chain.md

```diff
@@ -764,16 +731,49 @@ def compute_activation_exit_epoch(epoch: Epoch) -> Epoch:
     return Epoch(epoch + 1 + ACTIVATION_EXIT_DELAY)
 ```
 
-#### `bls_domain`
+#### `compute_bls_domain`
 
 ```python
-def bls_domain(domain_type: uint64, fork_version: bytes=b'\x00' * 4) -> int:
+def compute_bls_domain(domain_type: uint64, fork_version: bytes=b'\x00' * 4) -> int:
     """
     Return the BLS domain for the ``domain_type`` and ``fork_version``.
     """
     return bytes_to_int(int_to_bytes(domain_type, length=4) + fork_version)
 ```
 
+#### `validate_indexed_attestation`
```

Comment:
Really `validate_indexed_attestation` should refactored to be `is_valid_indexed_attestation` and put under "Predicates".

Reply:
disagree! see other comment